catalog/catalog-app/src/main/resources/features.xml

```diff
@@ -38,11 +38,18 @@
         <bundle>mvn:org.codice.thirdparty/gt-opengis/${opengis.bundle.version}</bundle>
     </feature>
 
-    <feature name="catalog-core" version="${project.version}"
+    <feature name="catalog-core-transform-api" install="manual" version="${project.version}"
```

Comment:
I don't think you need to add install="manual" to this feature.

Reply:
fixed